bin/hermes

```diff
@@ -34,15 +34,15 @@ def request_get(path):
     """
     response = requests.get(settings.hermes_server + path)
 
-    if response.status_code != requests.codes.ok:
+    if response.status_code != requests.codes.ok or len(response.content) == 0:
```

Comment:
I find it more idiomatic in python to use

`not response.content` vs `len(response.content) == 0`